sorter/src/test/java/sortpom/parameter/LineSeparatorParameterTest.java

```diff
@@ -18,46 +18,46 @@ public void lineSeparatorWithSomehtingElseShouldThrowException() throws Exceptio
 
         new PluginParametersBuilder()
                 .setEncoding("UTF-8")
-                .setFormatting("***", false, false)
+                .setFormatting("***", false, false, false)
```

Comment:
I see that the default value in the tests differ, sometimes it is true and sometimes false. I think that it should be true since that is the default value. On the other hand this does not need to be corrected, since the parameter may be moved from the setFormatting method. Did I mention that I usually do picky reviews of code? :-)